src/pool/rpc.h

```diff
@@ -131,15 +132,6 @@ CRT_RPC_DECLARE(pool_create, DAOS_ISEQ_POOL_CREATE, DAOS_OSEQ_POOL_CREATE)
 
 /* clang-format on */
 
-/* the source of pool map update operation */
-enum map_update_source {
-	MUS_SWIM = 0,
-	/* May need to differentiate from administrator/csum scrubber/nvme healthy monitor later.
-	 * Now all non-swim cases fall to DMG category.
-	 */
-	MUS_DMG,
-};
-
```

Comment:
sorry did not check it intime.
But why you remove this? this is I added that try to leave for future usage for rebuild's error handling to differentiate the source of trigger the rebuild.

Reply:
Current rebuild err handling need some refining, for example ds_pool_tgt_revert_rebuild() cannot do it for some cases, so I added the source to try to provide some more info to make it possible to refine the err handling in future.

Reply:
I was not aware of that, i thought it is not used anymore, let's add it back.